modules/api/src/main/java/net/countercraft/movecraft/craft/CraftType.java

```diff
@@ -125,9 +126,14 @@ public CraftType(File f) {
             Yaml yaml = new Yaml();
             data = (Map) yaml.load(input);
             input.close();
-        } catch (IOException e) {
+        }
+        catch (IOException e) {
             throw new TypeNotFoundException("No file found at path " + f.getAbsolutePath());
         }
+        catch (ScannerException e) {
+            e.printStackTrace();
+            throw new TypeParseException("Unable to parse " + f.getAbsolutePath());
+        }
```

Comment:
Instead of creating new exceptions, I recommend simply catching them at the top level. This should provide a better debugging experience to users.

Reply:
That makes a lot of sense